react/package.json

```diff
@@ -8,6 +8,7 @@
     "apollo-client": "^2.4.7",
     "classnames": "^2.2.6",
     "graphql": "^14.0.2",
+    "ramda": "^0.26.1",
```

Comment:
_Curiosity:_ Ramda is, now, already built-in in all of Render webpages. Still, adding it here makes because of IDE intelisense